01 - Community/People/jdanielmourao.md

```diff
@@ -38,11 +38,13 @@ publish: true
 - 
 -->
 
-<!--
 ## Sponsor this author
 
+- [[Buy me a coffee]]: https://ko-fi.com/jdanielmourao
```

Comment:
Don't forget to add the block IDs after!

Reply:
I've added them, I hope, in the right way?

Reply:
Yes, they look correct to me..
@argenos I think it might be worth changing the template bits that have block IDs to include `<>` placeholder for the URL - it might make it a bit clearer as to how to use the block IDs... maybe ...???